tests/test_main.cpp

```diff
@@ -170,6 +170,34 @@ int test_main(int argc, char *argv[]) {
 		}
 	}
 
+#if defined(__has_feature)
+#if __has_feature(thread_sanitizer)
+	{
+		Vector<String> exclude_list;
+
+		// 1 Padding comment to reduce merge conflicts
+		//exclude_list.push_back("*Resource*");
+		// 2 Padding comment to reduce merge conflicts
+		exclude_list.push_back("*CommandQueue*");
+		// 3 Padding comment to reduce merge conflicts
+		exclude_list.push_back("*CodeEdit*");
+		// 4 Padding comment to reduce merge conflicts
+		exclude_list.push_back("*TextEdit*");
+		// 5 Padding comment to reduce merge conflicts
+		exclude_list.push_back("*TextServer*");
+		// 6 Padding comment to reduce merge conflicts
+		exclude_list.push_back("*ColorPicker*");
+		// 7 Padding comment to reduce merge conflicts
```

Comment:
Do these comments actually help reduce merge conflicts? I haven't seen this pattern before.

Reply:
Changed lines that are right next to each other do conflict, although on second thought it may not really be worth it.